python/lsst/daf/persistence/posixStorage.py

```diff
@@ -102,20 +85,23 @@ def getMapperClass(root):
             if os.path.exists(os.path.join(basePath, "_parent")):
```

Comment:
I don't expect that this _parent code will go away any time soon. I think I need to add a test and impl code here for the new style cfgs (when a repo has a sparsely populated cfg and a parent cfg that defines its mapper)

I don't know about line that says `globals = {}`, I'm fairly certain it was copied from butler when I created the posixStorage class. I'll look into it though.